components/organisms/Repositories/repositories.tsx

```diff
@@ -37,7 +37,7 @@ const Repositories = ({ repositories }: RepositoriesProps): JSX.Element => {
     isError: repoListIsError,
     isLoading: repoListIsLoading,
     setPage,
-  } = useRepositories(repositories, Number(range), Number(limit));
+  } = useRepositories(repositories, Number(range ?? 30), Number(limit ?? 10));
```

Comment:
refactor (blocking): set the defaults where the variables are destructure on line 28

```diff
- const { pageId, toolName, selectedFilter, userOrg, range, limit } = router.query;
+ const { pageId, toolName, selectedFilter, userOrg, range = 30, limit = 10 } = router.query;
```

```suggestion
  } = useRepositories(repositories, Number(range), Number(limit));
```